iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/preprocessing/main/oozie_app/workflow.xml

```diff
@@ -119,6 +119,12 @@
 			<description>metadata extraction cache location, path pointing to
 				root cache directory holding meta.json file</description>
 		</property>
+		<property>
+            <name>referenceextraction_project_fundingclass_blacklist_regex</name>
+            <value>$^</value>
```

Comment:
In [an answer on SO](http://stackoverflow.com/a/940840/817499) they advise to use `a^` instead of `$^` because it's immune to multi-line strings, so maybe this is something that we should use?

Another thing is that a regexp pattern like this looks strange to me. Maybe an explicit comment in the description along the lines of "Note that $^ regexp pattern results in all projects being accepted." would be in order?